src/testing-utils/preTest.mjs

```diff
@@ -38,8 +38,10 @@ async function init() {
 
   const accounts = await getAccounts(web3)
 
-  const { ensAddress } = await deployTestEns({ web3, accounts })
+  const contracts = await deployTestEns({ web3, accounts })
```

Comment:
Can we change this variable name to something more appropriate, as it also includes  addresses of the owner etc